### ISSUE TITLE
Duplicate Code

### ISSUE THREAD

**Issue by HedgeComp** [mtniehaus/AutopilotBranding#15]
https://github.com/mtniehaus/AutopilotBranding/blob/51f11b5eba6b5f17835e08b244086a0ba4829ddf/AutopilotBranding/AutopilotBranding.ps1#L174C1-L179C2

**Comment by HedgeComp** [comment]
Opps .. Nevermind, I think I see what your doing now. Disabling WSUS , add or removing the Features and then re-enabling it.

I suggest adding another quick  comment block perhaps.

**Comment by andrewj-t** [comment]
Yep that is right. It is doing this so that Add-WindowsCapability command will be able to download the FODs from Windows Update directly - WSUS Servers don't support hosting the FOD content.

**Comment by HedgeComp** [comment]
Thanks for the clarification Andrew. I was thinking this was designed more for as a Windows Desktop script for using with Autopilot, so I didn't think of it as a script to use with Server operating systems. 

If WSUS is not applicable in Windows 11 , are you commenting  it out or leaving it in your running scripts. Watching SteveWs vid or Rubix it appears he keeps it but removes EU-V etc..

**Comment by andrewj-t** [comment]
WSUS can be applicable for clients as well, they may have a policy to get their updates from a WSUS server (which will break Add-WindosCapabality if it were not for the changes that the script does).

I started working on my Windows 11 24H2 image recently which is where I started using AutopilotBranding. I've just uploaded the most of the changes I made to github here - https://github.com/andrewj-t/AutopilotBranding/tree/win11-24h2-myconfig

In the PS script, I did comment out the UE-V section since we dont use that in our environment, and also added a registry key to the default user profile to left align the start menu by default. Everything else I could do via the config file

**Comment by mtniehaus** [comment]
I've added logic to make UE-V optional, and added additional logic to optionally left-align the start menu.  This will be available in the next version (soon).
